attalos/preprocessing/extract_vgg_features.py

```diff
@@ -0,0 +1,123 @@
+import sys
```

Comment:
There should be some attribution that this is from: https://github.com/karpathy/neuraltalk/blob/master/python_features/extract_features.py